package.json

```diff
@@ -29,7 +29,7 @@
     "codecov": "^2.2.0",
     "ember-ajax": "^3.0.0",
     "ember-async-button": "^1.0.2",
-    "ember-cli": "2.13.2",
+    "ember-cli": "~2.15.1",
```

Comment:
We need to upgrade the code as well if necessary.
https://github.com/ember-cli/ember-addon-output/compare/v2.13.2...v2.15.1